frontend/staff-dashboard/src/App.tsx

```diff
@@ -14,20 +13,50 @@ import {
 } from "components/layout";
 import LoginPage from "pages/login";
 import { DashboardPage } from "pages/dashboard";
+import { DiscountList, DiscountEdit, DiscountShow, DiscountCreate } from "pages/discounts";
+import axios from "axios";
+import useDrfDataProvider from "hooks/useDrfDataProvider";
 
-const {UserOutlined} = Icons;
+const {UserOutlined, BarcodeOutlined} = Icons;
+const axiosInterface = axios.create();
+
+axiosInterface.interceptors.request.use((config: any) => {
+  let token = sessionStorage.getItem(`oidc.user:${OIDC_CONFIG.authority}:${OIDC_CONFIG.client_id}`);
+
+  if (token !== null) {
+    token = JSON.parse(token).access_token;
+    config.headers.Authorization = `Bearer ${token}`;
+  }
+
+  return config;
+}, (error: any) => Promise.reject(error));
 
 export default function App() {
   const authProvider = useAuthProvider();
+  const xonlineProvider = useDrfDataProvider("http://mitxonline.odl.local:8013/api/v0");
```

Comment:
It looks like this is hardcoded to the local hostname, but we can figure out separately how to deal with this since we need it to work both locally with the live server and in deployed environments with the static build (once I get that working on heroku), could you open an issue for that?

Reply:
Added issue https://github.com/mitodl/mitxonline/issues/550 for this one.